Gir.toml

```diff
@@ -1962,3 +1961,8 @@ status = "generate"
     #value glib.Variant
     name = "action-target"
     ignore = true
+
+[[object]]
+name = "Gtk.TextViewLayer"
+status = "generate"
+version = "3.14"
```

Comment:
Please, move this between Gtk.TextView and Gtk.Toolbar.

Reply:
We should add a test to enforce that it's alpha sorted in the toml file. I'm going to add when I have a bit of time. :)